kubespawner/spawner.py

```diff
@@ -242,19 +237,8 @@ def _namespace_default(self):
                 return f.read().strip()
         return 'default'
 
-    ip = Unicode(
-        '0.0.0.0',
-        config=True,
-        help="""
-        The IP address (or hostname) the single-user server should listen on.
-
-        We override this from the parent so we can set a more sane default for
-        the Kubernetes setup.
-        """
-    )
-
     cmd = Command(
-        None,
+        ['jupyterhub-singleuser'],
```

Comment:
Is there a reason this default config is changed? It doesn't seem relevant to this PR.

Reply:
As long as you are fine with the changes here I can reverse this change.  I need 1 of the two.
https://github.com/jupyterhub/kubespawner/pull/386/files/f75a5790eff424d6db55872463b26691dce1a453#diff-3082573191d96b79e2ec71986d4ef3e2R1386